### DISCUSSION TITLE
DietPi-Software | Planka

### DISCUSSION THREAD

**Discussion by VasilevAndrei** [MichaIng/DietPi#6221] [Software Request]
Formal software information

- Software title | Planka
- Short description | Free open source kanban board for workgroups. (Trello alternative)
- Official URL (if available) | https://planka.app/
- GitHub URL (if available) | https://github.com/plankanban/planka
- (Official) install documentation | https://docs.planka.cloud/docs/installl-planka/Debian%20&%20Ubuntu

Are there similar/alternative software titles available with DietPi-Software?

- Nope

Of course you can run it in docker, but I think it would be cool to have this as one of the software options. 

![demo](https://user-images.githubusercontent.com/49864869/223425950-b45b5d72-5115-41ee-843c-7e68149be862.gif)

**Comment by Joulinar** [comment]
Please vote for this software implementation via ⬆️ button at first post.
